db/manual_compaction_test.cc

```diff
@@ -149,7 +177,121 @@ TEST_F(ManualCompactionTest, Test) {
 
   // close database
   delete db;
-  DestroyDB(dbname_, ROCKSDB_NAMESPACE::Options());
+  DestroyDB(dbname_, Options());
+}
+
+TEST_F(ManualCompactionTest, SkipLevel) {
+  DB* db;
+  Options options;
+  options.num_levels = 3;
+  options.level0_file_num_compaction_trigger = INT_MAX;
+  options.compaction_style = kCompactionStyleLevel;
+  options.create_if_missing = true;
+  options.compression = kNoCompression;
+  LogCompactionFilter* filter = new LogCompactionFilter();
+  options.compaction_filter = filter;
+  ASSERT_OK(DB::Open(options, dbname_, &db));
+
+  // Keys: 1, 2, 4, 8
+  // Each key is flushed to a new SST file in L0
+  for (int k = 1; k <= 8; k <<= 1) {
+    db->Put(WriteOptions(), std::to_string(k), "");
```

Comment:
We have a ToString() in string_util.h to use for portability.

Reply:
Refactored and got rid of `to_string`